### ISSUE TITLE
Default value for watch

### ISSUE THREAD

**Issue by lightonflux** [dalance/procs#36]
Version: procs 0.9.2

**Behaviour**:

```
$ procs -w
error: The argument '--watch <second>' requires a value but none was supplied
```

**Expectation**: 

Start watching immediately with a default value, but leave it to the user to specify a value if wanted. This behaviour is quite common with established linux/unix commands that support a watch mode, or *watch` itself.

**Comment by dalance** [comment]
Thank you for your suggestion.
I want to add the feature, but there is a problem.

For example, `procs -w 10` can be interpreted by the two ways.

* watch mode with default value and filter by "10"
* watch mode with 10s and no filter

I think there are some solution.
What do you think?

* Use `--`

```
$ procs -w 10  // watch mode with 10s
$ procs -w -- 10 // watch mode with  default value and filter by "10"
```

* `--watch-interval` option

```
$ procs -w  // watch mode with default value
$ procs -w --watch-interval 10 // watch mode with 10s
```

* `-w` and `-W`

```
$ procs -w  // watch mode with default value
$ procs -W 10 // watch mode with 10s
```

**Comment by lightonflux** [comment]
`--` is good for people already familiar with shell in greater detail. And i think you application aims also for users that might not be as familiar with these quirks and like something simple and easy to use. 

So i think having `-w` and `-W` is the best method. It is not the most beautiful solution, but the best middle ground for experienced and new users alike. 

About: `--watch-interval`. 
IMHO all options should have a long version. For documenting scrips later on.

So maybe 

`-w`, `--watch` → default
`-W [NUM]` , `--watch-interval [NUM]` → custom value

**Comment by dalance** [comment]
Thanks!
I agree `-w` and `-W` is best.
I released v0.9.6 which implements new options.
